Ecommerce/index.html

```diff
@@ -1,44 +1,46 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>E-commerce</title>
-    <meta charset="UTF-8" />
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <!-- Start coding from the lines below for ECOMMERCE TASK -->
-   
-    <!-- to import image from public folder, do it like this = <img src="../public/css.icon.png" alt="CSS image"> -->
-  <img src="../public/css.icon.png">
-  <ul>
-    <li> about</li>
-    <li> product</li>
-    <li> contact us</li>
-  </ul>
-  <h1>
-    About us
-  </h1>
-  <p>Welcome to my shop! We  are a one-shop destination for all your shopping needs . Our mission is to proide you with the best products and expceptional customer service. </p>
- <h1> Products</h1>
- <P>Explore the perfec5t items for yourself or a gift for  your loved ones.</P>
- <ul>
-  <li>Electronics</li>
```

Comment:
There's a typo in the opening tag of the paragraph. It should be &lt;p&gt; instead of &lt;P&gt;.